test/NuGet.Core.Tests/NuGet.Commands.Test/SpecValidationUtilityTests.cs

```diff
@@ -72,8 +81,19 @@ public void SpecValidationUtility_VerifyFrameworks_Unsupported()
 
             spec.AddProject(project);
 
-            // Act && Assert
-            AssertError(spec, "Invalid target framework");
+            // Act
+            try
+            {
+                SpecValidationUtility.ValidateDependencySpec(spec, new HashSet<string>(), mockLogger.Object);
```

Comment:
Try the `Should().Throw(...` FluentAssertion syntax for asserting thrown exceptions.